src/server.py

```diff
@@ -334,6 +334,9 @@ class HackSoundServer(Gio.Application):
         <method name='StopSound'>
           <arg type='s' name='uuid' direction='in'/>
         </method>
+        <method name='TermSound'>
```

Comment:
`TerminateSound` is more clear.

Reply:
Yeah SIGTERM was coined a long time ago. Nowadays is better to go with clear variable names.